orchagent/portsorch.cpp

```diff
@@ -24,10 +24,13 @@ PortsOrch::PortsOrch(DBConnector *db, vector<string> tableNames) :
 {
     SWSS_LOG_ENTER();
 
-    /* Initialize Counter Table */
+    /* Initialize counter table */
     DBConnector *counter_db = new DBConnector(COUNTERS_DB, "localhost", 6379, 0);
```

Comment:
Suggest manage 'new' resources by smart pointers (eg. std::unique_ptr). Otherwise, you should handle memory exceptions, clean up before ctor() throw, add dtor().